otter/test/convergence/test_gathering.py

```diff
@@ -419,145 +424,128 @@ def test_invalid_first_entry(self):
         self.assertRaises(ValueError, extract_CLB_drained_at, feed)
 
 
+def lb_req(method, url, json_response, response):
```

Comment:
Totally non-blocking - since this method returns a 200 every time, which is usually a response code to a GET request (since the PUTs and POSTs usually return 202 or 201 or something), can we just assume whatever calls this method is going to call it with a GET?